tee/prototype/epochproc.go

```diff
@@ -85,10 +85,12 @@ func (e *Enclave) txRoutine(
 		log = log.WithField("epoch", txEpoch.Number)
 	}
 
+	txCounter := 0
 	stagedTxs := makeTxCache()
 	for {
 		select {
 		case exiters := <-exits:
+			log.Infof("received %d TXs", txCounter)
```

Comment:
"txRoutine: processed %d TXs" so we know where this log comes from.

Reply:
I even deleted `txRoutine:` before I submitted cause I thought it was only necessary for `trace`.
:D